safe_autonomy_sims/pettingzoo/inspection/weighted_multi_inspection_v0.py

```diff
@@ -321,12 +325,13 @@ def step(
 
         # Store previous simulator state
         self.prev_state = self.sim_state.copy()
-        self.prev_num_inspected = (
-            self.chief.inspection_points.get_num_points_inspected()
-        )
-        self.prev_weight_inspected = (
-            self.chief.inspection_points.get_total_weight_inspected()
-        )
+        if self.simulator.sim_time > 0:
+            self.prev_num_inspected = (
+                self.chief.inspection_points.get_num_points_inspected()
+            )
+            self.prev_weight_inspected = (
+                self.chief.inspection_points.get_total_weight_inspected()
+            )
```

Comment:
This condition means anything inspected on initialization will be ignored. What is the CoRL behavior for this case?

Reply:
This check enables inspected points from initialization to be rewarded in the first step. please see my related [comment](https://github.com/act3-ace/safe-autonomy-sims/pull/41#discussion_r1889098091).